src/app/Fake.DotNet.Fsi/Fsi.fs

```diff
@@ -75,8 +75,10 @@ type FsiParams = {
 (* - LANGUAGE - *)
     /// Generate overflow checks
     Checked: bool option
-    /// Define a conditional compilation symbols
+    /// Define a conditional compilation symbol
```

Comment:
Might be possible to mark this record member Obsolete with a suggestion to use `Definitions`?

Though now that I think about it isn't there something about record member with obsolete triggering every time the record is extended using `with` expressions?

Reply:
Yes indeed, I think we can't do that. But we can adjust documentation.
Sad that this slipped through in the process of moving to fake 5 ;)

Reply:
I've added to the comment to suggest using the new property.